10-hash-table/HashTable/hashTable.c

```diff
@@ -0,0 +1,167 @@
+#include <stdbool.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "../list/list.h"
+
+void printHashTable(List* hashTable[], int *hashTableSize, bool *errorCode) {
+    printf("<Word> - <Frequency>\n");
+    for (int i = 0; i < *hashTableSize; ++i) {
+        if (hashTable[i] == NULL) {
+            continue;
+        }
+        Position j = next(first(hashTable[i], errorCode), errorCode);
+        while (j != NULL) {
+            printf("%s - %d\n", getValue(j, errorCode), getFrequency(j, errorCode));
+            j = next(j, errorCode);
+        }
+    }
+}
+
+int hashStringPolynomial(const char* word, const int tableSize) {
+    const int coefficientOfMultiplication = 29;
+    int polynomial = 0;
+    for (int i = 0; word[i] != '\0'; ++i) {
+        const unsigned int numberInAlphabet = word[i];
+        polynomial = (polynomial * coefficientOfMultiplication + numberInAlphabet) % tableSize;
+    }
+    return polynomial;
+}
+
+bool isSpecialCharacters(const char symbol) {
```

Comment:
```suggestion
bool isSpecialCharacter(const char symbol) {
```
Или isOneOfSpecialCharacters. Речь же про symbol